file-source/README.adoc

```diff
@@ -0,0 +1,33 @@
+= Spring Cloud Stream File Sink
```

Comment:
??? Is something wrong here?
I wonder if we should tell here about `File Source` instead...